dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb

```diff
@@ -210,7 +210,7 @@ jenkins:
                   key: "JAVA_HOME"
                   value: "<%= agent['javaHome'] %>"
               - envVar:
-                  key: "PATH+JAVA_HOME"
+                  key: "PATH.JAVA_HOME"
```

Comment:
```suggestion
                  key: "PATH_JAVA_HOME"
```

Reply:
The idea is to avoid the confusion with the _ of java_home, it's probably not the definitive solution but at least the dot is accepted by the regexp that make the check : 
```
Invalid value: "PATH+JAVA_HOME": a valid environment variable name must consist of alphabetic characters, digits, '_', '-', or '.', and must not start with a digit (e.g. 'my.env-name',  or 'MY_ENV.NAME',  or 'MyEnvName1', regex used for validation is '[-._a-zA-Z][-._a-zA-Z0-9]*')
```